scalikejdbc-core/src/main/scala/scalikejdbc/ConnectionPoolSettings.scala

```diff
@@ -10,5 +10,6 @@ case class ConnectionPoolSettings(
   validationQuery: String = null,
   connectionPoolFactoryName: String = null,
   driverName: String = null,
-  warmUpTime: Long = 100L)
+  warmUpTime: Long = 100L,
+  timeZone: String = null)
```

Comment:
@tkawachi I can understand your opinion in general, but let's go with nullable String here to keep API compatibility with `validationQuery`, `connectionPoolFactoryName` and `driverName`.